### ISSUE TITLE
How to configure webhooks on my forks?

### ISSUE THREAD

**Issue by evandrocoan** [1egoman/backstroke#59]
On the StackOverflow question:

1. https://stackoverflow.com/questions/28639713/how-to-automatically-align-sync-a-forked-github-origin-master-branch-to-upstream

> > It is very easy to sync forked repository but question is how ?

> In this example i am using [WorldMapGenerator Repository  ](http://github.com/kevalbhatt/WorldMapGenerator)
 
> >  Go to your forked repository, you can see setting button click on it.

> [![enter image description here][1]][1]

> > After clicking on setting button you can see Webhooks & services option in left menu click on it.

[![enter image description here][2]][2]

> > Then you can see *Add Webhook Button* on right side just click on it.

> [![enter image description here][3]][3]

> > After clicking Add Webhook Button, details page will get open, so in detail page yo can see *payload url*. Enter http://backstroke.us url in it.

> [![enter image description here][4]][4]

> Now If any commit goes in main repository, then pull request will come in your forked repository.

> That's it enjoy :)

> For more details https://github.com/1egoman/backstroke

  [1]: http://i.stack.imgur.com/GyX4R.png
  [2]: http://i.stack.imgur.com/VaWqb.png
  [3]: http://i.stack.imgur.com/0L7nY.png
  [4]: http://i.stack.imgur.com/OzJdH.png

They show these steps, but I did not understand if the are correct. When he says `Enter http://backstroke.us url in it`, it is the `https://backstroke.us/_5a4ds65f46464s65d4654` URL, or just `http://backstroke.us`?

If I do this in all my forks, I will get automatically updates? I do not need creates links and generate the URLs `https://backstroke.us/_5a4ds65f46464s65d4654` for all my forks?

**Comment by 1egoman** [comment]
@evandrocoan There are two versions of Backstroke:

1. [Backstroke Classic](https://github.com/1egoman/backstroke#use-backstroke-classic), the original version. It was made originally by me about a year ago and is deprecated, however others have posted about it online. In order to set this up, the user adds `https://backstroke.us` as a webhook in their repository manually.

2. [Backstroke](https://github.com/1egoman/backstroke#add-backstroke-to-a-repository), the newer version of Backstroke. This has a dashboard the user can use to setup Backstroke, no manually-adding of webhooks required. The url `https://backstroke.us/_5a4ds65f46464s65d4654` is the webhook that gets added behind the scenes to your repository, but you never have to call it yourself unless you want to tie Backstroke into some custom system. If you just want to sync changes from one repo to another, then you don't have to think about the webhook since it's managed by the system.

TL;DR: Use Backstroke. At some point in the future it's my plan to get rid of Backstroke classic, though it won't happen without a lot of advance warning.
